gammapy/cube/simulate.py

```diff
@@ -176,3 +176,37 @@ def sample_background(self, dataset):
         table.rename_column("DEC_TRUE", "DEC")
 
         return EventList(table)
+
+    def sample_psf(self, psf_map, events):
+        """Sample psf map.
+
+        Parameters
+        ----------
+        psf_map : `PSFMap`
```

Comment:
You need to give full links to resolve, in this case:
```
~gammapy.cube.PSFMap
```
and below
```
~gammapy.data.EventList
```

Reply:
Done